README.md

```diff
@@ -1,4 +1,6 @@
-# seatunnel [![Build Status](https://travis-ci.org/InterestingLab/seatunnel.svg?branch=master)](https://travis-ci.org/InterestingLab/seatunnel)
+# seatunnel
+
+![example workflow](https://github.com/InterestingLab/seatunnel/actions/workflows/backend.yml/badge.svg?branch=wd-v2-baseline)
```

Comment:
Should the name be changed to `Backend`

Reply:
nop, I think we need github action file name here, instead of name we configure in yml file. I test in my env, and find it work for  now
![image](https://user-images.githubusercontent.com/15820530/142171792-efe0ef7b-1a19-49e2-908f-0ee423ad711e.png)

Reply:
![image](https://user-images.githubusercontent.com/16631152/142337644-57cdb295-6cda-41ec-885b-96d9288899a5.png)
yup, but example always feels informal

Reply:
My bad, I will change the name.